app/components/sidebars/main/main_sidebar.js

```diff
@@ -255,11 +257,11 @@ export default class ChannelSidebar extends Component {
     onPageSelected = (index) => {
         this.swiperIndex = index;
 
-        if (this.refs.drawer) {
+        if (this.drawerRef?.current) {
             if (this.swiperIndex === 0) {
-                this.refs.drawer.canClose = false;
+                this.drawerRef.current.canClose = false;
```

Comment:
Haven't used the null coalescing operator enough, but can it be used on assignment as well?
```js
this.drawerRef?.current.canClose = this.swiperIndex !== 0
```

Reply:
it seems that eslint complains about this, but will simplify it anyway
![image](https://user-images.githubusercontent.com/6757047/57778122-64b09200-76f1-11e9-826c-e72097f24509.png)